app/src/main/java/com/concordia/mcga/fragments/NavigationFragment.java

```diff
@@ -68,8 +107,42 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         //Init Fragments
         transportButtonFragment = (TransportButtonFragment) getChildFragmentManager().findFragmentById(R.id.transportButton);
         indoorMapFragment = (IndoorMapFragment) getChildFragmentManager().findFragmentById(R.id.indoormap);
-
         //Init View Components
+        navigationSearch = (EditText) parentLayout.findViewById(R.id.navigationSearch);
+        mapCenterButton = (FloatingActionButton) parentLayout.findViewById(R.id.mapCenterButton);
+        mapCenterButton.setOnClickListener(new OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Log.d("Testing mapCenterButton", "Initializing OnClickListener");
+                if (!gpsmanager.isProviderEnabled(LocationManager.GPS_PROVIDER)) {
+                    Log.d("Testing AlertGPS Launch", "Initializing method");
```

Comment:
These are mostly log remnants from when I was testing an enormous amount of methods. The pull request was done before I could clear them. Most of them are unnecessary and will be removed.